test/Maxfork/MaxFork.c

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang %s -emit-llvm %O0opt -c -o %t1.bc
+// RUN: %klee --max-forks=1 %t1.bc
```

Comment:
The test doesn't have any oracles.  As it is, it wouldn't detect any bugs in the implementation of this feature.   Even better, think whether you have some checks that would fail w/o your change.